src/store/wizardSlice.ts

```diff
@@ -44,6 +56,13 @@ const initialState: wizardState = {
     accountType: 'google',
     email: undefined,
   },
+  // @ts-ignore
+  profile: undefined,
+  openScap: {
+    profileId: 'None',
```

Comment:
the `None` profileId shouldn't be sent to the backend, it would be better to have undefined there too.